### ISSUE TITLE
HTML in Markdown disappearing when line breaks missing

### ISSUE THREAD

**Issue by johnpuddephatt** [tightenco/jigsaw#318]
Having problems with HTML in Markdown not rendering properly if it isn't well formatted. Basically it seems like multiple sibling elements on one line is the problem – only the first element makes it through.

e.g. this is fine:
```
<h1>This works</h1>
<p>Some text</p>
```
but only the first tag from the below makes it through:
```
<h1>This doesn't</h1><p>Some text</p>
```

**Comment by damiani** [comment]
This appears to be an issue in Parsedown (https://github.com/erusev/parsedown-extra/issues/44) dating back to 2015, which has been fixed but only in a beta release. I'll take a closer look and see if we would have any issues using that beta.

**Comment by hellocosmin** [comment]
That beta would also make build-time code syntax highlighting possible (https://github.com/erusev/parsedown/issues/568) 👍

**Comment by damiani** [comment]
Oooh that's slick. I recall having issues last year trying the 1.8 beta, but it's evolved a bit since the last time I looked into it.

**Comment by anthonyterrell** [comment]
Closed by #330
